### DISCUSSION TITLE
Fuji PXR and motorized control valve for gas supply to Probat LG3 burner

### DISCUSSION THREAD

**Discussion by Broederen** [artisan-roaster-scope/artisan#740] [General]
Can anyone help with the programming of the PRX4 and selection of the valve? The valve is only for flowcontrol to the burner on my Probat LG3. Safety is obtain by selenoid valve (closed when not activated). Gas supply through inox pipe with diameter of 10mm.. Apart from the possibility to control through the PXR4 I also like to have the 'manual' possibility. I have original UG15 levers which i would like to utilise, means translation of handle position into position of the controlvalve. Help is much appreciated.

**Comment by roasterdave** [comment]
This question fits better as a Discussion since it is not a problem report for Artisan.  This issue will be converted to a Discussion and the issue closed.
